helm/README.md

```diff
@@ -0,0 +1,113 @@
+# Stargate deployment using Helm
+Want to deploy Stargate on a Kubernetes cluster using Helm? Follow the instructions below. 
+
+## Pre-requisites
+
+You'll need a Cassandra cluster running with the storage port 7000 accessible as a headless service, and make sure [Helm](https://helm.sh) is available in the environment.
+
+### Cassandra installation
+For a quick start, you can install Cassandra in your Kubernetes cluster using Helm.
+
+```shell script
+ helm install my-release bitnami/cassandra
+```
+
+### Autoscaling
+Autoscaling uses metrics server. Metrics server can be installed by executing the command:
```

Comment:
Does the Autoscaler scale all types of pods including Coordinator and all API Services? Would be good to note which types of pods will be scaled.